roles/power_aix_bootstrap/tasks/python_install.yml

```diff
@@ -1,33 +1,34 @@
 # Copyright (c) IBM Corporation 2020
 ---
 # CHECK for Yum on inventory host
-  - name: Check for existence of yum
-    raw: "rpm -q --quiet yum && echo true || echo false"
-    register: yum_exists
+- name: Check for existence of yum
+  raw: "rpm -q --quiet yum && echo true || echo false"
+  register: yum_exists
+  changed_when: false
```

Comment:
Why does the changed_when as added here?

Reply:
the `raw` module will always report `changed` even tho in this particular task nothing will every changes, it is merely querying for the existence of yum in the lpar. hence, it is not appropriate to report `changed` when nothing will ever change. so the role linting CI flagged this and this is how you fix it